api-compatibility.versions

```diff
@@ -1,2 +1,2 @@
 stargate_version=v2.1.0-BETA-12
-data_api_version=v1.0.10
+data_api_version=v1.0.11
```

Comment:
The latest version is v1.0.12. Can we upgrade to the same?

Reply:
@kathirsvn done in 5e2231e. Tested locally and tests pass with data-api v1.0.12